src/moodleapi/Moodleapi.ts

```diff
@@ -0,0 +1,39 @@
+interface Response {
```

Comment:
Shoving this here for lack of a better place: If we create a `moodleapi/index.ts` with:

```javascript
export * from './Moodleapi'
```

We can then import the class as just:

```javascript
import { MoodleApi } from './moodleapi'
```

:point_up: provides some parity with the import conventions on `eventsapi`, but is a minor subjective nicety.